setup.py

```diff
@@ -18,6 +18,7 @@
     "torch",
     "sentencepiece",
     "scipy",
+    "intel_extension_for_pytorch"
```

Comment:
Should we move it to `EXTRAS_REQUIRE` to keep the openvino / neural-compressor subpackages independent from IPEX ?

Reply:
Done ✔️